watertap/examples/flowsheets/ion_exchange/tests/test_ion_exchange_demo.py

```diff
@@ -138,19 +139,97 @@ def test_specific_operating_conditions(self, ix_0D):
             )
         ) == pytest.approx(0.000211438, rel=1e-3)
 
-        assert value(m.fs.ion_exchange.number_columns) == pytest.approx(4, rel=1e-3)
-        assert value(m.fs.ion_exchange.bed_depth) == pytest.approx(1.7, rel=1e-3)
-        assert value(m.fs.ion_exchange.t_breakthru) == pytest.approx(
-            56759.75759, rel=1e-3
-        )
-        assert value(m.fs.ion_exchange.partition_ratio) == pytest.approx(
-            235.99899, rel=1e-3
-        )
+        results_dict = {
```

Comment:
This is pretty unsightly and could get cleaned up. That is why we use approx after all, I think I include 4 significant figures and pytest to 1e-3. This applies to a lot of code across the test files.

Reply:
Ask and ye shall receive.